manual/tracy.tex

```diff
@@ -1692,6 +1692,16 @@ \subsubsection{OpenCL}
 
 Similar to Vulkan and OpenGL, you also need to periodically collect the OpenCL events using the \texttt{TracyCLCollect(ctx)} macro. An excellent place to perform this operation is after a \texttt{clFinish} since this will ensure that any previously queued OpenCL commands will have finished by this point.
 
+\subsubsection{CUDA}
+
+CUDA support is enabled by including the \texttt{public/tracy/TracyCUDA.hpp} header file. Tracing CUDA requires the creation of a Tracy CUDA context using the macro \texttt{TracyCUDAContext()}, which returns an instance of a \texttt{TracyCUDACtx} object. This context must be used when profiling CUDA events. Cleanup is handled using the \texttt{TracyCUDAContextDestroy(ctx)} macro. To assign a custom name to the context, use the \texttt{TracyCUDAContextName(ctx, name, size)} macro.
```

Comment:
there's no `TracyCUDACtx` object, it's `tracy::CUDACtx`

Reply:
We should mention that using `TracyCUDA.hpp` requires the "NVIDIA CUPTI library" that ships with the "NVIDIA CUDA Toolkit", and that it is located at `[cuda-installation-path]/extras/CUPTI`

Reply:
TracyCUDA only allows for a single CUDACtx object at any given time. Subsequent calls to TracyCUDAContext will simply return the same reference-counted object. There is no need for clients to instantiate multiple CUDACtx objects since the CUDACtx is capable of instrumenting all CUDA contexts and streams.